src/nft/components/profile/view/FilterSidebar.tsx

```diff
@@ -200,10 +220,12 @@ const CollectionSelect = ({
       </Box>
       <Box paddingBottom="12" borderRadius="8">
         <Column as="ul" paddingLeft="0" gap="10" style={{ maxHeight: '80vh' }}>
-          <CollectionFilterSearch
-            collectionSearchText={collectionSearchText}
-            setCollectionSearchText={setCollectionSearchText}
-          />
+          {!hideSearch && (
```

Comment:
we should also hide when loading so users don't see the search bar while collections are loading in and then it gets removed

Reply:
great catch thanks done.